### PR TITLE
DDF-1410: Update to have a unique list of actions displayed.

### PR DESCRIPTION
**Pull request by ckurata**
@ricklarsen, @stustison, @djblue, please review this pull request.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/108)

### CODE REVIEW

File: catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/SearchController.java [hunk 1]
```diff
@@ -498,6 +498,7 @@ private JSONArray getActions(Metacard metacard) {
         List<Action> actions = actionRegistry.list(metacard);
         for (Action action : actions) {
             JSONObject actionJson = new JSONObject();
+            actionJson.put(Search.ACTIONS_ID, action.getId() + action.getTitle());
             actionJson.put(Search.ACTIONS_ID, action.getId());
```
Comment (stustison) [inline catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/SearchController.java]: Did you forget to remove the other "put" into this map with the same key as the one that you just added?

**Comment by ckurata** [comment]
@stustison: Yes, I forgot to remove the original line. It is fixed now. Thank you.

**Comment by stustison** [comment]
:+1:

**Comment by ricklarsen** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/108)

**Comment by djblue** [comment]
I was able to manually verify the changes without any issues.

**Pull request merged**